src/components/action-list/pages/action-list.jsx

```diff
@@ -82,31 +82,34 @@ const ActionLists = () =>
     <DocsBlock info='Space between'>
       <ActionList direction={DIRECTION.SPACE_BETWEEN}>
         <ActionListHole>
-          <Label iconType={ICON_TYPE.ANSWER} iconColor={ICON_COLOR.GRAY_SECONDARY} secondary={true}
+          <Label iconType={ICON_TYPE.ANSWER} iconColor={ICON_COLOR.GRAY_SECONDARY} secondary
             size={LABEL_SIZE.SMALL}>
             <Text size={TEXT_SIZE.OBSCURE} weight={TEXT_WEIGHT.BOLD} color={TEXT_COLOR.GRAY_SECONDARY}
-              type={TEXT_TYPE.DIV}>0/5</Text>
+              type={TEXT_TYPE.DIV}>0/5
+            </Text>
           </Label>
         </ActionListHole>
         <ActionListHole>
-          <Label iconType={ICON_TYPE.COUNTER} iconColor={ICON_COLOR.GRAY_SECONDARY} secondary={true}
+          <Label iconType={ICON_TYPE.COUNTER} iconColor={ICON_COLOR.GRAY_SECONDARY} secondary
             size={LABEL_SIZE.SMALL}>
             <Text size={TEXT_SIZE.OBSCURE} weight={TEXT_WEIGHT.BOLD} color={TEXT_COLOR.GRAY_SECONDARY}
-              type={TEXT_TYPE.DIV}>2d: 00h</Text>
+              type={TEXT_TYPE.DIV}>2d: 00h
+            </Text>
           </Label>
         </ActionListHole>
-        <ActionListHole><ButtonSecondary type={BUTTON_SECONDARY_TYPE.ALT_INVERSE} small={true}>
-          start
-        </ButtonSecondary>
+        <ActionListHole>
+          <ButtonSecondary type={BUTTON_SECONDARY_TYPE.ALT_INVERSE} small>
+            start
+          </ButtonSecondary>
         </ActionListHole>
       </ActionList>
     </DocsBlock>
 
     <DocsBlock info='No wrap' multiContent={[
-      <ContrastBox narrow={true}>
-        <ActionList noWrap={true}>
+      <ContrastBox key={1} narrow>
```

Comment:
why here key?
writing keys is on DocBlack side

Reply:
Elements in arrays need to have a key